umbra-js/src/classes/Umbra.ts

```diff
@@ -403,7 +403,7 @@ export class Umbra {
    * @dev Return type is an ethers Signature: { r: string; s: string; _vs: string, recoveryParam: number; v: number; }
    * @param spendingPrivateKey Receiver's spending private key that is doing the signing
    * @param chainId Chain ID where contract is deployed
-   * @param version Umbra contract version
+   * @param contract Umbra contract address that will accept transaction
```

Comment:
Nit on phrasing here
```suggestion
   * @param contract Umbra contract address that withdrawal transaction will be sent to
```